src/nvim/shada.c

```diff
@@ -3482,6 +3489,8 @@ static inline char *get_converted_string(const vimconv_T *const sd_conv,
           (tgt) = aetv.vval.v_list; \
         } \
       } while (0)
+// Workaround for silencing linter.
+#define ELSE else
```

Comment:
@justinmk `//` inside a macros?! And for `CHECKED_KEY` this will look like

```C
  else if (CHECK_KEY( /* NOLINT(readability/braces) */ \
      unpacked.data.via.map.ptr[i].key, name)) { \
```

: otherwise there is not enough space.